src/App.tsx

```diff
@@ -16,80 +17,149 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+  NONE = 'None',
+  ALPHABET = 'Alphabet',
+  LENGTH = 'Length',
 }
 
 type ReorderOptions = {
   sortType: SortType,
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  switch (sortType) {
+    case ('Alphabet'):
```

Comment:
d'int rely on unknown string, use enum value instead 
```suggestion
    case SortType.ALPHABET:
```